src/dotnet/commands/dotnet-list/dotnet-list-p2ps/Program.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.DotNet.Tools.List.ProjectToProjectReferences
+{
+    public class ListProjectToProjectReferencesCommand
+    {
+        public static int Run(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            CommandLineApplication app = new CommandLineApplication(throwOnUnexpectedArg: false)
+            {
+                Name = "dotnet list p2ps",
```

Comment:
we would have to change it everywhere then - none of the commands localizes it - it is simply command invokation which IMO should not be localized:
https://github.com/dotnet/cli/blob/rel/1.0.0/src/dotnet/commands/dotnet-build/Program.cs#L18
https://github.com/dotnet/cli/blob/rel/1.0.0/src/dotnet/commands/dotnet-migrate/Program.cs#L23
https://github.com/dotnet/cli/blob/rel/1.0.0/src/dotnet/commands/dotnet-test/Program.cs#L22